src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,147 @@
+package duke;
+
+import duke.exception.DukeException;
+import duke.processor.TaskList;
+import duke.task.Task;
+
+/**
+ * Class that represents the duke.Ui, used to print
+ * statements for the bot.
+ *
+ * @author Melissa Anastasia Harijanto
+ */
+public class Ui {
+    /** String that represents a line. */
+    protected final String line = "____________________________________________________________";
+
+    /**
+     * Greets the user.
+     */
+    public void greet() {
+        String greetings = "\nHello! I'm Duke\n" + "What can I do for you?\n";
+        System.out.println(line + greetings + line);
+    }
+
+    /**
+     * Prints a message when the user adds a task to the bot.
+     *
+     * @param task The task that is added.
+     * @param amountOfTasks The total amount of tasks that has been added by the user.
+     */
+    public void addTask(Task task, int amountOfTasks) {
+        System.out.println(line);
+        System.out.println("Got it. I've added this task:");
+        System.out.println(task.toString());
+        System.out.println("Now you have " + amountOfTasks + " tasks in the list.");
+        System.out.println(line);
+
+    }
+
+    /**
+     * Prints the list of tasks added by the user.
+     *
+     * @param tasks The list of tasks to be printed.
+     */
+    public void list(TaskList tasks) {
```

Comment:
Hmm, I would say this is fine but maybe we can be more explicit with the naming(I know, really knitty gritty, would say
this is fine honestly), say maybe name it listOut(..)?